site/en/docs/extensions/mv3/messaging/index.md

```diff
@@ -289,57 +283,59 @@ Extensions [can exchange messages][native-client] with native applications that
 
 ### Content scripts are less trustworthy {: #content-scripts-are-less-trustworthy }
 
-[Content scripts are less trustworthy][security-cs] than the extension service worker (for example, a malicious web
-page might be able to compromise the renderer process where the content scripts run). Assume that
-messages from a content script might have been crafted by an attacker and make sure to [validate and
-sanitize all input][security-santize]. Assume any data sent to the content script might leak to the web page.
+[Content scripts are less trustworthy][security-cs] than the extension service worker.
+For example, a malicious webpage might be able to compromise the rendering process that
+runs the content scripts. Assume that messages from a content script might have been
+crafted by an attacker and make sure to [validate and sanitize all input][security-santize].
+Assume any data sent to the content script might leak to the web page.
 Limit the scope of privileged actions that can be triggered by messages received from content
 scripts.
 
 ### Cross-site scripting {: #cross-site-scripting }
 
-When receiving a message from a content script or another extension, your scripts should be careful
-not to fall victim to [cross-site scripting][wiki-cross-site]. This advice applies to scripts running inside the
-extension background page as well as to content scripts running inside other web origins.
-Specifically, avoid using dangerous APIs such as the ones below:
+When receiving a message from a content script or another extension, make sure to protect
+your scripts against [cross-site scripting][wiki-cross-site]. This advice applies to scripts running inside the
+extension background page as well as to content scripts running in other web origins. Use APIs that don't run
+scripts whenever possible:
 
 {% Label %}content-script.js or service-worker.js{% endLabel %}
```

Comment:
@wizardofdocs can you remove `content-script.js` from labels of code samples that include `chrome.tabs.sendMessage()`. This can only be used by the service worker or scripts in extension pages (HTML pages that are part of the extension)
```suggestion
{% Label %}service-worker.js{% endLabel %}
```

Reply:
Fixed. Thanks!